i_scene_cp77_gltf/__init__.py

```diff
@@ -1,6 +1,6 @@
 bl_info = {
     "name": "Cyberpunk 2077 glTF Importer",
-    "author": "HitmanHimself, Turk, Jato, dragonzkiller, kwekmaster, glitchered",
+    "author": "HitmanHimself, Turk, Jato, dragonzkiller, kwek/kwekmaster, glitchered",
```

Comment:
Remove this. We're also going to be changing this soon anyway.

Reply:
kk will change

Reply:
changed